src/Notifications/NotificationCenterController.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Dynamo.Controls;
+using Dynamo.Notifications.View;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
+
+namespace Dynamo.Notifications
+{
+    public class NotificationCenterController
+    {
+        NotificationUI notificationUIPopup;
+        public NotificationCenterController(DynamoView dynamoView, Button notificationsButton)
+        {
+            notificationUIPopup = new NotificationUI();
+            notificationUIPopup.IsOpen = false;
+            notificationUIPopup.PlacementTarget = notificationsButton;
+            notificationUIPopup.Placement = PlacementMode.Bottom;
+            notificationUIPopup.HorizontalOffset = -285;
+            notificationUIPopup.VerticalOffset = 10;
+
+            dynamoView.SizeChanged += DynamoView_SizeChanged;
+            dynamoView.LocationChanged += DynamoView_LocationChanged;
+            notificationsButton.Click += NotificationsButton_Click;
+
+            notificationUIPopup.webView.EnsureCoreWebView2Async();
+            notificationUIPopup.webView.CoreWebView2InitializationCompleted += WebView_CoreWebView2InitializationCompleted;            
+        }
+
+        private void WebView_CoreWebView2InitializationCompleted(object sender, Microsoft.Web.WebView2.Core.CoreWebView2InitializationCompletedEventArgs e)
+        {
+            var assembly = Assembly.GetExecutingAssembly();
+            var htmlFile = "Dynamo.Notifications.Web.index.html";
+            var fontFile = "Dynamo.Notifications.Web.ArtifaktElement-Regular.woff";
+            var jsFile = "Dynamo.Notifications.Web.index.bundle.js";
```

Comment:
Can these be exported to resource file? Or are these temporary for testing only?

Reply:
I think these names should be fixed and they don't need localization. Maybe define as static readonly strings